browser_wasm.go

```diff
@@ -50,11 +50,11 @@ func CreateWindow(_, _ int, title string, monitor *Monitor, share *Window) (*Win
 	height := js.Global().Get("innerHeight").Int()
 
 	devicePixelRatio := js.Global().Get("devicePixelRatio").Float()
-	canvas.Set("width", int(float64(width)*devicePixelRatio+0.5))   // Nearest non-negative int.
-	canvas.Set("height", int(float64(height)*devicePixelRatio+0.5)) // Nearest non-negative int.
+	canvas.Set("width", width)
+	canvas.Set("height", height)
 
 	canvas.Get("style").Call("setProperty", "width", "100vw")
-	canvas.Get("style").Call("setProperty", "height", "100vh")
+	canvas.Get("style").Call("setProperty", "height", "100dvh")
```

Comment:
Unfortunately, `100vh` didn't account for the user agent's user interface (e.g. address bar) height, and using [`dvh`](https://www.w3.org/TR/css-values-4/#dynamic-viewport-percentage-units) was the simplest solution I could find.